mappings/net/minecraft/world/BiomeView.mapping

```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/class_5423 net/minecraft/world/BiomeView
```

Comment:
Hum, I think this class is more like a "RegistryReadyWorldView" but other people have probably better ideas